### DISCUSSION TITLE
MARKED FOR DELETION

### DISCUSSION THREAD

**Discussion by Adam-Mould** [saleor/saleor#8755] [Show and tell]
This discussion has been marked for deletion

**Comment by karolkielecki** [comment]
@Adam-Mould Great work! 👏👏👏

**Comment by zaiste** [comment]
@Adam-Mould it looks great! Congrats on the release!

Here at Saleor, we are currently working on standardising the process of building frontend applications that integrate with the Saleor API. Our first-attempt is [React Storefront](https://github.com/saleor/react-storefront), an open-source starter kit  providing solid defaults when building storefronts with React.js & Next.js.

I've seen you used Next.js in your project. Maybe there would be a way to combine our experiences? Ultimately we want to provide an easy-to-use starting point for quickly creating storefronts on top of Saleor.
